java/src/main/java/org/wildfly/openssl/OpenSSLEngine.java

```diff
@@ -28,11 +32,8 @@
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import javax.net.ssl.SSLEngine;
-import javax.net.ssl.SSLEngineResult;
-import javax.net.ssl.SSLException;
-import javax.net.ssl.SSLParameters;
-import javax.net.ssl.SSLSession;
+
+import static org.wildfly.openssl.Messages.MESSAGES;
 
```

Comment:
Reordering imports, although seemingly harmless, can lead to merge conflicts.

Reply:
Most of the java classes are already organized imports alphabetically. Retaining this standard for this file as well.